keyform/templates/keyform/includes/searchtable.html

```diff
@@ -41,7 +41,7 @@
                 </div>
                 <div class="row">
                     <div class="large-4 columns">
-                       {% trans 'Staff Name' %} <input type="text" name="staff_name" value="{{ search_data.staff_name }}"/>
+                       {% trans 'Staff' %} <input type="text" name="staff" value="{{ search_data.staff }}"/>
```

Comment:
We should display the staff's full name here instead of just their username:
[```{{ search_data.staff.get_full_name }}```](https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.get_full_name)
While you're at it, you might as well change it everywhere else we display the staff or author:
```bash
egrep --color=always -Inr "\.staff|\.author"
```